crates/bevy_render/src/lib.rs

```diff
@@ -473,35 +479,15 @@ unsafe fn initialize_render_app(app: &mut App) {
                     render_system,
                 )
                     .in_set(RenderSet::Render),
-                World::clear_entities.in_set(RenderSet::PostCleanup),
+                despawn_temporary_render_entity.in_set(RenderSet::PostCleanup),
```

Comment:
```suggestion
                despawn_temporary_render_entities.in_set(RenderSet::PostCleanup),
```
Need to rename the system definition too ofc